cmd/vicadmin/logs.go

```diff
@@ -273,8 +276,10 @@ func tarEntries(readers map[string]entryReader, out io.Writer) error {
 
 		// be explicit about the number of bytes to copy as the log files will likely
 		// be written to during this exercise
-		_, err = io.CopyN(t, e, e.Size())
-		_ = e.Close()
+		if e != nil {
+			e.Close()
+			_, err = io.CopyN(t, e, sz)
```

Comment:
This is inverting the Close/Copy ordering from previously - is that intentional?